src/views/collection-view/components/collection-meta-progress/helpers/parse-collection-progress.ts

```diff
@@ -0,0 +1,155 @@
+import { Collection, TutorialLite } from 'lib/learn-client/types'
+import { ApiCollectionTutorialProgress } from 'lib/learn-client/api/api-types'
+import { getTutorialSlug } from 'views/collection-view/helpers'
+
+/**
+ * Given progress data for a particular collection,
+ * and a count of all tutorials in that collection,
+ * Return booleans representing collection progress state.
+ */
+function parseCollectionProgress(
+	progressData: ApiCollectionTutorialProgress[],
+	tutorials: TutorialLite[],
+	collection: Pick<Collection, 'id' | 'slug'>
+) {
+	/**
+	 * The basics
+	 */
+	const tutorialCount = tutorials.length
+	const completedTutorialCount = countCompletedRecords(
+		progressData || [],
+		collection.id
+	)
+	const isCompleted = completedTutorialCount == tutorialCount
+	const isInProgress = completedTutorialCount > 0 && !isCompleted
```

Comment:
Currently on learn, when at least 1 collection is in progress we show the progress bar with '0/x'. So we may want to adjust this a bit if we want to retain that pattern. 
<img width="1044" alt="Screen Shot 2022-09-14 at 9 42 38 AM" src="https://user-images.githubusercontent.com/36613477/190213678-509b6cfe-cdae-4881-8d55-9d938b2c3c6d.png">

Reply:
✅ ah nice, thanks for the callout - have updated the logic for `isInProgress`, so this should be working now!